tools/coqdep_common.ml

```diff
@@ -476,16 +483,24 @@ let mL_dependencies () =
 
 let coq_dependencies () =
   List.iter
-    (fun (name,_) ->
+    (fun (name,absname) ->
        let ename = escape name in
        let glob = if !option_noglob then "" else " "^ename^".glob" in
-       printf "%s%s%s %s.v.beautified: %s.v" ename !suffixe glob ename ename;
-       traite_fichier_Coq !suffixe true (name ^ ".v");
-       printf "\n";
-       printf "%s.vio: %s.v" ename ename;
-       traite_fichier_Coq ".vio" true (name ^ ".v");
-       printf "\n";
-       flush stdout)
+       let out_ch =
+         if StrSet.mem absname !vfileOutputs then
```

Comment:
I think it might be better to make `coq_dependencies` simply take a boolean argument, determining for all files whether to output to stdout or to write .v.d files, and pass `!create_file_output` here, rather than in `treat_file`?